admin_guide/manage_nodes.adoc

```diff
@@ -280,7 +280,12 @@ $ oc delete node <node>
 ----
 $ oc get nodes
 ----
-+
+
+[NOTE]
+====
+`$ oc delete node <node>` command does not clean the filesystem of a node. A node can register itself if you do not perform the xref:../install/uninstalling_cluster.adoc#uninstalling-nodes[uninstallation of a node].
```

Comment:
* s/`$ oc delete node <node>` command/The `$ oc delete node <node>` command
* As a naive reader, it's not clear what it means for a node to "register itself". Do you mean a node that you have performed `$ oc delete node <node>` on? Register itself with OpenShift?
* Consider "if you do not xref ...blabla.adoc[uninstall the node].

Reply:
Addressed